src/calculateRentalCost.js

```diff
@@ -20,7 +20,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
```

Comment:
you can move this operation to the variable `days * 40`

also, create a variable for car cost.